WalletWasabi/Services/CcjClient.cs

```diff
@@ -358,7 +358,8 @@ private async Task TryProcessRoundStateAsync(long ongoingRoundId)
 				var mineCount = myOutputs.Count(x => x.Value == denomPair.value);
 
 				Money denomination = denomPair.value;
-				Money expectedCoordinatorFee = denomination.Percentage(ongoingRound.State.CoordinatorFeePercent * denomPair.count);
+				int anonset = Math.Min(110, denomPair.count); // https://github.com/zkSNACKs/WalletWasabi/issues/1379
+				Money expectedCoordinatorFee = denomination.Percentage(ongoingRound.State.CoordinatorFeePercent * anonset);
```

Comment:
`ongoingRound.State.CoordinatorFeePercent` is configurable and doesn't have an upper cap. Should worth to consider also apply a limit to that value on the client-side, otherwise the server could set it to 100%.

I think the client could also have a hard limit, some kind of "client will never sign any transaction that pays more than 0.00xxbtc"

Reply:
I remember having this issue long time ago, but that should be fixed. I think with a 1% check somewhere.  

Anyway, we could check these parameters when we receive them from the server so we won't refuse to sign, but don't even register into the round in the first place?

Reply:
Yes, I think that's the best.

Reply:
Actually while I was implementing it I noticed we have a better solution already implemented:

```cs
// Every time the user enqueues (intentionally writes in password) then the coordinator fee percent must be noted and dequeue later if changes.
CcjClientRound latestRound = State.GetLatestRoundOrDefault();
CoordinatorFeepercentToCheck = latestRound?.State?.CoordinatorFeePercent;
```

This makes more sense.

Reply:
You can see my progress here, but I throw it out, because the network fee sanity checks are controversial. Who are we to say that $100 is too expensive for network fees? We can at best set network sanity checks for a half a year period and if they would go up too much then we'd be fucked, because it'd break every client and the volume would be zero. Dangerous to set any limit there.

https://github.com/nopara73/WalletWasabi/tree/sanitychecks